dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemServiceImpl.java

```diff
@@ -117,15 +121,19 @@ public void delete(Context context, RequestItem requestItem) {
     }
 
     @Override
-    public boolean isRestricted(Context context, DSpaceObject o)
-            throws SQLException {
-        List<ResourcePolicy> policies = authorizeService
-                .getPoliciesActionFilter(context, o, Constants.READ);
+    public boolean isRestricted(Context context, DSpaceObject o) throws SQLException {
+        List<ResourcePolicy> policies = authorizeService.getPoliciesActionFilter(context, o, Constants.READ);
+
         for (ResourcePolicy rp : policies) {
-            if (resourcePolicyService.isDateValid(rp)) {
+            if (rp.getEPerson() != null && !rp.getEPerson().equals(context.getCurrentUser())
+                || !groupService.isMember(context, rp.getGroup())) {
+                continue;
+            }
+            if (!List.of("restricted", "reserved").contains(rp.getRpName()) && resourcePolicyService.isDateValid(rp)) {
```

Comment:
restricted and reserved are special names used in some customization but not apply in general in dspace